architecture/additional_concepts/authorization.adoc

```diff
@@ -424,47 +435,33 @@ administrator may still specify a `RunAsUser` if they wish.
 
-The admission controller is aware of certain conditions in the security context constraints that trigger
-it to look up pre-allocated values from a namespace and populate the security context constraint
-before processing the pod.
-
-The following security context constraints prompt the admission controller to look for pre-allocated
-values:
-
-. A RunAsUser strategy of MustRunAsRange with no min/max set.  Admission will look for the `openshift.io/sa.scc.uid-range`
-annotation to populate range fields.
-. An SELinux strategy of MustRunAs with no level set.  Admission will look for the `openshift.io/sa.scc.mcs`
-annotation to populate the level.
-. A FSGroup strategy of MustRunAsRange with no ranges set.  Admission will look for the
-`openshift.io/sa.scc.supplemental-groups` annotation.
-. A SupplementalGroups strategy of MustRunAsRange with no ranges set.  Admission will look for
-the `openshift.io/sa.scc.supplemental-groups` annotation.
-During the generation phase, the security context provider will default any values that are not
-specifically set by the user.  Defaulting is based on the strategy being used:
-.. RunAsAny and MustRunAsNonRoot strategies do not provide default values
-.. MustRunAs (single value) strategies provide the value that must be used
-.. MustRunAsRange and MustRunAs (range based) strategies provide the min value of the range.
-If a range based strategy is configurable with multiple ranges it will provide the min value of
-the first configured range.
+The admission controller is aware of certain conditions in the security context constraints that trigger it to look up pre-allocated values from a namespace and populate the security context constraint before processing the pod. Each SCC strategy is evaluted indpendently of other strategies, with the pre-allocated (default) values for each policy aggregated to make the final values for the various ids defined in the pod.
+
+The following security context constraints cause the admission controller to look for pre-allocated values:
+
+. A *RunAsUser* strategy of `MustRunAsRange` with no min/max set.  Admission will look for the "openshift.io/sa.scc.uid-range" annotation to populate range fields.
+. An *SELinuxContext* strategy of `MustRunAs` with no level set.  Admission will look for the "openshift.io/sa.scc.mcs" annotation to populate the level.
+. A *FSGroup* strategy of `MustRunAs`.  Admission will look for the "openshift.io/sa.scc.supplemental-groups" annotation.
+. A *SupplementalGroups* strategy of `MustRunAs`.  Admission will look for the "openshift.io/sa.scc.supplemental-groups" annotation.
+
+During the generation phase, the security context provider will default any values that are not specifically set in the pod.  Defaulting is based on the strategy being used:
+
+. *RunAsAny* and *MustRunAsNonRoot* strategies do not provide default values. Thus, if the pod needs a group id this id must be defined in the pod spec.
```

Comment:
It is true for all fields using the two strategies mentioned.  So, since these bullets are talking specifically about defaulting it can read `*RunAsAny* and *MustRunAsNonRoot* strategies do not provide default values. Thus, if the pod needs a specific value in a field governed by these strategies it must be set in the pod spec`.

Since these strategies do not supply defaults namespace values do not apply.  

In the bullets below it is accurate to say generically where the default will come from (either configured in the strategy itself or via namespace configuration if the strategy supports it - we detail which strategies support allocated values elsewhere)

Hope that makes sense.  What I'm really going for is avoiding talking about field specific behaviors when talking about the strategies in a general sense.